### PR TITLE
to Support ARM

### PR DESCRIPTION
**Pull request by zhangxinmrs**
## CHANGES PROPOSED:
-  
-  

## ISSUES ADDRESSED:
- Closes 

## TESTS PERFORMED:
-
-

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by zhangxinmrs** [comment]
support ARM

**Comment by Kitzunu** [comment]
Please fillout the template and fix all errors

**Comment by Winfidonarleyan** [comment]
Many errors, when there is time, I will write more

File: deps/mersennetwister/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+add_library(mersennetwister INTERFACE)
```
Comment (Winfidonarleyan) [inline deps/mersennetwister/CMakeLists.txt:1]: Need add license

File: src/cmake/macros/FindMySQL.cmake [hunk 2]
```diff
@@ -12,77 +12,6 @@
 
 set( MYSQL_FOUND 0 )
 
-if(WIN32)
```
Comment (Winfidonarleyan) [inline src/cmake/macros/FindMySQL.cmake:15]: what?

File: src/cmake/showoptions.cmake [hunk 3]
```diff
@@ -65,6 +79,13 @@ else()
   message("* Use coreside debug              : No  (default)")
 endif()
 
+if( MARIADB )
```
Comment (Winfidonarleyan) [inline src/cmake/showoptions.cmake:82]: what?

**Pull request closed without merging**